webpack/assets/javascripts/foreman_tools.js

```diff
@@ -97,6 +97,11 @@ export function updateTable(element) {
   if (perPage !== undefined) {
     uri.setSearch('per_page', perPage);
   }
+
+  pageNum = $('#pagination1-page').val();
```

Comment:
i might be missing something, but where does the pagination1-page comes from? why is it called "1"?